content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -324,7 +324,7 @@ init container completion record has been lost due to garbage collection. This
 applies for Kubernetes v1.20 and later. If you are using an earlier version of
 Kubernetes, consult the documentation for the version you are using.
 
-## {{% heading "whatsnext" %}}
+## What's next {#whats-next}
```

Comment:
```markdown
## {{% heading "whatsnext" %}}
```
is correct @chrisnegus 

Any incoming hyperlinks should be fixed to work with the existing shortcode.

Reply:
The shortcode lets us use a consistent heading throughout the site. It also lays the ground for custom navigation, eg we could make the page intro render differently for pages that have a “What's next” section.

Reply:
Okay. So the link pointing to it is wrong. I'll fix that. Thanks!

Reply:
Sorry, I should have checked that the` ## {{% heading "whatsnext" %}}` heading is the standard way of identifying the What's next sections. I fixed the broken link to that section on the page as well. Please let me know if I did that properly.